src/cryptography/hazmat/primitives/serialization/pkcs7.py

```diff
@@ -263,6 +263,87 @@ def encrypt(
         return rust_pkcs7.encrypt_and_serialize(self, encoding, options)
 
 
+def pkcs7_decrypt_der(
+    data: bytes,
+    certificate: x509.Certificate,
+    private_key: rsa.RSAPrivateKey,
+    options: typing.Iterable[PKCS7Options],
+) -> bytes:
+    return _pkcs7_decrypt(
+        data, certificate, private_key, options, serialization.Encoding.DER
+    )
+
+
+def pkcs7_decrypt_pem(
+    data: bytes,
+    certificate: x509.Certificate,
+    private_key: rsa.RSAPrivateKey,
+    options: typing.Iterable[PKCS7Options],
+) -> bytes:
+    return _pkcs7_decrypt(
+        data, certificate, private_key, options, serialization.Encoding.PEM
+    )
```

Comment:
I don't love needing to pass an encoding down, ideally we'd be able to PEM decode the data and then simply call `pkcs7_decrypt_der` with the results. That's probably annoying to do in Python though.

Reply:
A few solutions:
- Separating the PEM decode logic in another rust function, and calling it separately
- Finding a library that does PEM to DER in Python
- Coding it in-house (but probably annoying and / or dirty)

I'm for the 1st one, I think.

Reply:
I've implemented a quick function in python using rust to decode pem to der. Please let me know what you think!